### ISSUE TITLE
Random tab stop at the end of breadcrumbs

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#14730]
Tabbing your way through things: Filter -> Explore -> Some element in breadcrumbs -> People

Should be (maybe) Filter -> Explore -> First breadcrumbs item (or all of breadcrumbs) -> Favourites (sticky header)

**Comment by turt2live** [comment]
I'm told this is expected and I forgot (as a mouse user) that I clicked things.
